packages/jest-cli/src/index.ts

```diff
@@ -5,21 +5,4 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-// TODO: remove @jest/core exports for the next major
-import {
-  SearchSource,
-  TestScheduler,
-  TestWatcher,
-  getVersion,
-  runCLI,
-} from '@jest/core';
-import {run} from './cli';
-
-export = {
-  SearchSource,
-  TestScheduler,
-  TestWatcher,
-  getVersion,
-  run,
-  runCLI,
```

Comment:
um, so we don't export these anymore, why?

Reply:
not from `jest-cli`, only from `@jest/core`